File: ejercicios-js/ejercicio2.js
console.log("ejercicio 2:verificador de par/impar")
function verificarParidad() {
    // Generar un número aleatorio entre 0 y 99
    const num = Math.floor(Math.random() * 100);

    // mostrar el número generado
    console.log(`Número generado: ${num}`);

    // Verificar si es par o impar y mostrar el resultado
    console.log(num % 2 === 0 ? "✅ Es par" : "❌ Es impar");
}

// Llamar a la función para probarla
verificarParidad();

File: ejercicios-py/ejercicio2.py
import random

def verificar_par_impar():
    numero = random.randint(1, 100)
    if numero % 2 == 0:
        resultado = "par"
    else:
        resultado = "impar"
    return f"El número generado es {numero} y es {resultado}."

# Ejemplo de uso
print(verificar_par_impar())